app/assets/stylesheets/administrate/components/_field-unit.scss

```diff
@@ -0,0 +1,17 @@
+.field-unit {
+  @include clearfix;
+  @include fill-parent;
+  align-items: center;
+  display: flex;
+  margin-bottom: $base-spacing;
+  position: relative;
+}
+
+.field-unit__label {
+  @include span-columns(2 of 12);
```

Comment:
Do you think some sort of flex-basis on the label would be more appropriate here, and just let the `.field-unit__field` fill with width?